discovery_engine_core/core/src/storage/sqlite.rs

```diff
@@ -759,6 +758,82 @@ impl FeedbackScope for SqliteStorage {
         tx.commit().await?;
         Ok(document)
     }
+
+    async fn update_time_spend(
+        &self,
+        document: document::Id,
+        view_mode: ViewMode,
+        duration: Duration,
+    ) -> Result<TimeSpendDocumentView, Error> {
+        const MS_PER_DAY: i64 = 1000 * 60 * 60 * 24 * 7;
```

Comment:
🔧 this value represents milliseconds per week, but not per day

Reply:
oh yes I changed it back and force, I will change the value